### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by hanjk0302**

### CODE REVIEW

File: DataScience/readme.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Aufgabe
+It is a Coding Challenge for the Data Science Field. You can use one of them and Information in parentheses must not be communicated to candidates.
```
Comment (pietbrauer) [inline DataScience/readme.md:2]: > Information in parentheses must not be communicated to candidates

Nice idea but this is public for everyone to see so please remove the information the candidate shouldn't see 😉

File: DataScience/readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Aufgabe
```
Comment (pietbrauer) [inline DataScience/readme.md:1]: Please rephrase that in English

**Review by pietbrauer — CHANGES_REQUESTED** [review]

File: DataScience/readme.md (hunk 2 above)
Comment (hanjk0302) [inline DataScience/readme.md:1]: Coding Challenge

File: DataScience/readme.md (hunk 1 above)
Comment (hanjk0302) [inline DataScience/readme.md:2]: It is a Coding Challenge for the Data Science Field. You can use one of them.

File: DataScience/readme.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+It is a Coding Challenge for the Data Science Field. You can use one of them and Information in parentheses must not be communicated to candidates.
+
+## House prices
+How can you find your dream house in America and which elements influence the house price?  
+
+* Task (Medium): Analyse the data and find correlations. 
+(List of Analysis, Outliers, Nan value handle, Frequencies, interpretation of correlation etc. The list should not be given to candidate). 
+* Task (Medium): Feature engineering
+* Task (Challenging): predict house quality using proper variables (to try different models would be bonus when we evaluate.)
```
Comment (hanjk0302) [inline DataScience/readme.md:10]: * Task (Challenging): predict house quality using proper variables.

File: DataScience/readme.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* Task (Challenging): optimize the model (Hyperparameter tuning and fine tuning)
```
Comment (hanjk0302) [inline DataScience/readme.md:11]: * Task (Challenging): optimize the model.

File: DataScience/readme.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+* Task (Very Challenging): Use more features, maybe even add external data, for house quality prediction
+
+## Job Prediction
+How can you find a job using a given data?
+
+* Task (Easy): Analyse the data correlations (List of Analysis, Outliers, Nan value handle, Frequencies, interpretation of correlation etc. The list should not be given to candidate)
```
Comment (hanjk0302) [inline DataScience/readme.md:17]: * Task (Easy): Analyse the data and find correlations.

File: DataScience/readme.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+* Task (Medium): predict a job using proper variables (to try different models would be bonus when we evaluate.)
```
Comment (hanjk0302) [inline DataScience/readme.md:18]: * Task (Medium): predict a job using proper variables.

File: DataScience/readme.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Task (Challenging): optimize the model (Hyperparameter tuning and fine tuning)
```
Comment (hanjk0302) [inline DataScience/readme.md:19]: * Task (Challenging): optimize the model.

**Pull request merged**